package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-pay-statement-receiver",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Could argue this is 1.1.0 as the service now has a new feature of using the cache.

Reply:
Agreed -- properly bumped